python/python/core.c

```diff
@@ -1,77 +0,0 @@
-/* rizin - LGPL - Copyright 2016-2019 - pancake */
-
-#include "core.h"
-
-RzCore *core = NULL;
-
-/* TODO : move into a struct stored in the plugin struct */
-static void *py_core_call_cb = NULL;
-
-static int py_core_call(void *user, const char *str) {
-	if (py_core_call_cb) {
-		PyObject *arglist = Py_BuildValue ("(z)", str);
-		PyObject *result = PyEval_CallObject (py_core_call_cb, arglist);
-		const char * str_res = NULL;
-		if (result) {
-			if (PyLong_Check (result)) {
-				return (int)PyLong_AsLong (result);
-			} else if (PyLong_Check (result)) {
-				return (int)PyLong_AsLong (result);
-			} else if (PyUnicode_Check (result)) {
-				int n = PyUnicode_KIND (result);
-				switch (n) {
-				case 1:
-					str_res = (char*)PyUnicode_1BYTE_DATA (result);
-					break;
-				case 2:
-					str_res = (char*)PyUnicode_2BYTE_DATA (result);
-					break;
-				case 4:
-				default:
-					str_res = (char*)PyUnicode_4BYTE_DATA (result);
-					break;
-				}
-			} else
-			if (PyUnicode_Check (result)) {
-				str_res = PyBytes_AS_STRING (result);
-			}
-			if (str_res) {
-				rz_cons_print (str_res);
-				return 1;
-			}
-		}
-	}
-	return 0;
-}
-
-void Rizin_plugin_core_free(RzCorePlugin *ap) {
-	free ((char *)ap->name);
-	free ((char *)ap->license);
-	free ((char *)ap->desc);
-	free (ap);
-}
-
-PyObject *Rizin_plugin_core(Rizin* self, PyObject *args) {
```

Comment:
there's a reference to this function in python.c remove that as well.

Reply:
Removed that.